graphql/schemabuilder/pagination.go

```diff
@@ -401,35 +402,38 @@ type SafeBatchNodesToKeep struct {
 	mux         sync.Mutex
 }
 
-func (c *connectionContext) applyBatchTextFilter(ctx context.Context, nodes []interface{}, matchStrings []string, batchedFields map[string]*graphql.Field, nodesToKeep []bool) error {
-	g, ctx := errgroup.WithContext(ctx)
+func (c *connectionContext) applyBatchTextFilter(ctx context.Context, nodes []interface{}, matchStrings []string, batchedFields map[string]*graphql.Field, nodesToKeep []bool) []*graphql.ExecutionUnit {
 	m := &sync.Mutex{}
+	executors := []*graphql.ExecutionUnit{}
```

Comment:
nit: maybe call this `executionUnits`?  There's another struct called an executor, so naming this executors is a little confusing.